### ISSUE TITLE
dotnet process crashes when specifying stylecop.json additional file

### ISSUE THREAD

**Issue by powellcj12** [dotnet/cli#6334]
I'm working on integrating [StyleCop.Analyzers](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/Configuration.md) into my .NET Core project. I've managed to correctly get the ruleset file included in the build by specifying it in the ```CodeAnalysisRuleSet``` tag. Now I'm trying to add the JSON configuration file, but when I attempt to build it seems like the dotnet process just crashes.

## Steps to reproduce

1. Create stylecop.json file in the same location as the csproj file.
2. Add the following to the csproj file:
```XML
  <ItemGroup>
    <AdditionalFiles Include="stylecop.json" />
  </ItemGroup>
```
3. Run ```dotnet build```

## Expected  behavior
Build succeeds (though possible with warning/errors from StyleCop)

## Actual behavior
```
/usr/local/share/dotnet/sdk/1.0.1/Roslyn/Microsoft.CSharp.Core.targets(71,5): error MSB6006: "/usr/local/share/dotnet/sdk/1.0.1/Roslyn/Run
Csc.sh" exited with code 134.
```

Digging a bit deeper, I find this in the verbose output:
```
/usr/local/share/dotnet/sdk/1.0.1/Roslyn/RunCsc.sh: line 17: 17552 Abort trap: 6           "$DIR/../../../dotnet" "$DIR/csc.exe" "$@"
```

I do see that the ```/additionalfile``` parameter is being passed to ```csc.exe```:

```
... /analyzer:/Users/Charlie/.nuget/packages/stylecop.analyzers/1.0.1/analyzers/dotnet/cs/StyleCop.Analyzers.dll /additionalfile:stylecop.json ...
```

I also get an Apple crash dialog: [dotnet_crash.txt](https://github.com/dotnet/cli/files/924388/dotnet_crash.txt)

## Environment data

```
$ dotnet --info
.NET Command Line Tools (1.0.1)

Product Information:
 Version:            1.0.1
 Commit SHA-1 hash:  005db40cd1

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/1.0.1
```

**Comment by livarcocc** [comment]
@jaredpar @agocke  can you help us understand why csc is failing in this case?

**Comment by powellcj12** [comment]
I dug into this a bit more, and it appears to only happen when I specifically include the ```stylecop.json``` file. I can include other things like ```stylecop.txt``` and ```foo.json``` so I wonder if it has something to do with the analyzer itself?

**Comment by powellcj12** [comment]
@vweijsters suspects this is an issue with JSON parsing in the analyzers (closest issue seems like dotnetanalyzers/stylecopanalyzers#2290)... that said, it would still be nice if the dotnet process itself didn't completely blow up when this happens.

**Comment by powellcj12** [comment]
This seems to be resolved when I move to .NET Core 2 Preview 1 that was just released??!?

**Comment by powellcj12** [comment]
I installed .NET Core 2 Preview 1 today and didn't see this issue. Then I went back to v1.0.1 and didn't see it either... I wonder if StyleCopAnalyzers updated something on their end?

**Comment by powellcj12** [comment]
Ahh, all I did was change the version in my csproj files - I was still using .NET Core v2.0.0 CLI tools (and presumably SDK). When I revert back to using v1.0.1 (which wasn't easy to figure out), I still repro the issue. So presumably fixed in v2.0.0?

**Comment by livarcocc** [comment]
Looks like it. Closing the issue for now. If you can still repro this in 2.0, please re-activate.
